crates/matrix-sdk-crypto/src/verification/cache.rs

```diff
@@ -40,19 +40,47 @@ impl VerificationCache {
         self.verification.iter().all(|m| m.is_empty())
     }
 
+    /// Add a new `Verification` object to the cache, this will cancel any
+    /// duplicates we have going on with a given user.
```

Comment:
this doesn't say that it will also _cancel_ the new incoming request though, if there is a cancelled one.

Reply:
It does further bellow in the code, the method isn't publicly exposed but I added this fact to the doc comment in https://github.com/matrix-org/matrix-rust-sdk/pull/727/commits/bf91d1c79f14d7782105ece9b7caa65a67502c0c anyways.